src/interpolation/compute/dot.ts

```diff
@@ -0,0 +1,32 @@
+import { Formatters } from '../../types'
+
+export default (
+  path: string,
+  data: any,
+  options?: { formatters?: Formatters; fallback?: string }
+): any => {
+
+  const iterative: string[] = []
+  path.split('.').map(subPath => {
+    const selector = subPath.match(/\[.*\]/)
```

Comment:
I think there could be a bug here:

If I use `truc.bid[0]ule.machin`, I think it will be interpreted as `truc.bidule[0].machin`. It shouldn't be allowed

Reply:
Will get this fixed, thanks.

Reply:
https://github.com/Errorname/google-docs-mustaches/issues/16